### DISCUSSION TITLE
Why i have blur image

### DISCUSSION THREAD

**Discussion by genuineswe** [zoontek/react-native-bootsplash#644] [General]
hello admin, i have blur image like below,
1. 
![Screenshot_2024-10-22-15-54-45-249-edit_com](https://github.com/user-attachments/assets/ca71cd11-5a91-4d52-ab69-adb3bdf88d9f)
this is first navigation bar after click open the app, the image shows blurry
2. 
![Screenshot_2024-10-22-15-55-09-398-edit_com](https://github.com/user-attachments/assets/8afedcad-16e3-420c-88dd-6956c7ce9297)
then the navigation bar changing like this(not execute the .hide() yet). i have clear image

what happens here? 

thank you in advance

**Comment by zoontek** [comment]
@gaweki That's because it uses the [Android SplashScreen API](https://developer.android.com/develop/ui/views/launch/splash-screen#how).

Since Android 12, to improve (in a fake way) app launch time, the system uses some styles attributes ([windowSplashScreenBackground](https://developer.android.com/reference/android/R.attr#windowSplashScreenBackground), [windowSplashScreenAnimatedIcon](https://developer.android.com/reference/android/R.attr#windowSplashScreenAnimatedIcon), etc) to display a compressed, low on memory splash screen until your app **really** launch:

<img width="944" alt="Screenshot 2024-10-22 at 11 39 48" src="https://github.com/user-attachments/assets/7d11e1c5-275f-4af6-be5e-e8e2292eb3c7">

That's also why `react-native-bootsplash` has limited customization options: more complex stuff are not possible with the system handled part of the splash screen (background with a gradient, full screen image, etc).
